reference/6/Microsoft.PowerShell.Management/Add-Content.md

```diff
@@ -115,16 +119,17 @@ Add-Content -Path .\NewFile.txt -Value (Get-Content -Path .\CopyFromFile.txt)
 Get-Content -Path .\NewFile.txt
 ```
 
-The `Add-Content` cmdlet uses the **Path** and **Value** parameters to create a new file in the
-current directory. The **Value** parameter uses the `Get-Content` cmdlet to get the contents of an
-existing file, CopyFromFile.txt. The parentheses around the `Get-Content` cmdlet ensure that the
-command finishes before the `Add-Content` command begins. The **Value** parameter passes the
-content to `Add-Content` which updates the NewFile.txt file. The `Get-Content` cmdlet displays the
-contents of the new file, NewFile.txt.
+- The `Add-Content` cmdlet uses the **Path** and **Value** parameters to create a new file in the
+  current directory.
+- The `Get-Content` cmdlet gets the contents of an existing file, `CopyFromFile.txt`
+  and passes it to the **Value** parameter.
+  - The parentheses around the `Get-Content` cmdlet ensure that the command finishes before the
```

Comment:
I don't think this should be a sub-item. Just combine it with the parent sentence to form a two-sentence paragraph in one list item.